zip-0320.rst

```diff
@@ -203,9 +202,12 @@ will be replaced by:
   such that a Unified Address MUST **either**:
 
   * contain at least one shielded Receiver (Typecodes :math:`\mathtt{0x02}`
-    and :math:`\mathtt{0x03}`), and no Traceable P2PKH Receiver (Typecode
-    :math:`\mathtt{0x04}`); **or**
-  * contain **only** a Traceable P2PKH Receiver (Typecode :math:`\mathtt{0x04}`).
+    and :math:`\mathtt{0x03}`), and **no** Source Restriction Metadata Item
```

Comment:
```suggestion
    and :math:`\mathtt{0x03}`, or any future shielded Receiver typecode), and **no** Source Restriction Metadata Item
```
or otherwise adjust the wording to not appear to require that it can only be one of those two specific receiver codes, and never any future shielded Receiver typecode.

Reply:
The previous wording specified exactly these two typecodes, so this is preserving that restriction; an additional change to that part of the specification is out of scope for this ZIP.